.gitignore

```diff
@@ -7,6 +7,7 @@ models/
 # Local Chroma db
 .chroma/
 db/
+dbs/
```

Comment:
Maybe better to add `.gitignore` & `.gitkeep` inside the `dbs` folder and remove this from project `.gitignore`? An example of .gitignore:
```txt
*
!.gitignore
!.gitkeep
```

Reply:
I would agree, but at the moment I'm trying to follow the current layout of the project which added the original `db` folder to the gitignore, so I'm mirroring this.
I think a project structure refactor would probably be best done in a separate set of commits to simplify things.